File: mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int height, i, j, k;
    do
    {
        height = get_int("(1-8) Enter the height= ");
    }
    while (height < 1 || height > 8);
    for (i = 0; i < height; i++)
    {
        for (j = 0; j < height - (i + 1); j++)
        {
            printf(" "); // spaces
        }
        for (k = 0; k < i + 1; k++)
        {
            printf("#"); // backslashes
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if 1 <= h <= 8:
        break

for i in range(h):
    for k in range(h-(i+1)):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print()
